mos_tests/functions/common.py

```diff
@@ -301,10 +301,12 @@ def delete_instance(nova_client, uid):
             sleep(1)
 
 
-def create_instance(nova_client, inst_name, flavor_id, net_id, security_groups,
-                    image_id='', block_device_mapping=None, timeout=5):
+def create_instance(nova_client, inst_list, inst_name, flavor_id, net_id,
```

Comment:
Did you check all tests that uses this function?
And why we need such parameter in common function?
Why test cannot manage this list by test itself?